### PR TITLE
feat(terraform): add CKV NCP rules about Network ACL.

### PR DESCRIPTION
**Pull request by Floodnut**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description
Add terraform scan rules about provider Naver Cloud Platform.
These rules is associated with Network ACL.

- CKV_NCP_8 is associated with FTP 20 Port.
- CKV_NCP_9 is associated with FTP 21 Port.
- CKV_NCP_10 is associated with SSH 22 Port.
- CKV_NCP_11 is associated with RDP 3389 Port.

### Description
https://registry.terraform.io/providers/NaverCloudPlatform/ncloud/latest/docs/resources/network_acl_rule

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/terraform/checks/resource/ncp/NACLInboundCheck.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        for inbound in conf.get('inbound', []):
+            if inbound['rule_action'] == ["ALLOW"]:
+                ip = inbound.get('ip_block', '0.0.0.0/0')
+                if ip == ['0.0.0.0/0'] or ip == ['::/0']:
+                    port = inbound.get('port_range', str(self.port))[0]
+                    if port == str(self.port):
+                        return CheckResult.FAILED
+                    elif port.find('-'):
+                        portRange = list(map(int, port.split("-")))
+                        if portRange[0] <= self.port <= portRange[-1]:
```
Comment (gruebel) [inline checkov/terraform/checks/resource/ncp/NACLInboundCheck.py:25]:
```suggestion
                        port_range = list(map(int, port.split("-")))
                        if port_range[0] <= self.port <= port_range[-1]:
```
snake_case 🐍  please 🙂

**Review by gruebel — COMMENTED** [review]
nice, great work, added a couple of comments

**Comment by Floodnut** [comment]
I adjust comment!

**Review by gruebel — APPROVED** [review]
thanks for the hard work 🥇

**Review by nimrodkor — APPROVED** [review]
TY @Floodnut !

**Pull request merged**